source/custom/nidcpower_service.custom.cpp

```diff
@@ -1,5 +1,56 @@
 #include <nidcpower/nidcpower_service.h>
 
+#include <stdexcept>
+
 namespace nidcpower_grpc {
 
+struct DriverErrorException : std::runtime_error {
+  DriverErrorException(int status) : std::runtime_error("") { status_ = status; }
+  int status_ = 0;
+};
+
+void CheckStatus(int status)
+{
+  if (status != 0) {
+    throw DriverErrorException(status);
+  }
+}
+
+::grpc::Status NiDCPowerService::MeasureMultiple(::grpc::ServerContext* context, const MeasureMultipleRequest* request, MeasureMultipleResponse* response)
+  {
```

Comment:
Are you planning to add unit/system test for the MeasureMultiple API?

Reply:
@doshirohan , The tests are part of different task. So mostly not part of this PR

Reply:
@doshirohan  But we can visit once on how to add unit tests for this and if this has similar problems like the ones in FetchMultiple case, then it can be discussed earlier

Reply:
> if this has similar problems like the ones in FetchMultiple case, then it can be discussed earlier
I didn't write unit tests for FetchMultiple. I think what you are referring to is issue with respect to writing unit tests for niFake APIs. I had to write those because I updated mako scripts to support new data type and we wanted to have unit tests for those types. This is not related to FetchMultiple API itself.

Reply:
Oh ok.

Reply:
Tests can be a part of different PR.